src/management-system-v2/lib/useFuzySearch.ts

```diff
@@ -22,8 +22,7 @@ export default function useFuzySearch<TData extends Record<string, any>>(
   const searchParams = useSearchParams();
 
   const [searchQuery, setSearchQuery] = useState(
-    searchParams.get(fuzzySearchOptions.useSearchParams ? fuzzySearchOptions.queryName : '') ??
-      undefined,
+    searchParams.get(fuzzySearchOptions.useSearchParams ? fuzzySearchOptions.queryName : ''),
```

Comment:
I think I changed this to convert `null` to `undefined`, because some components allow only `undefined` for optional properties. E.g. antd Input.value

Is there a reason you want to keep `null`?

Reply:
I didn't think about that, I just saw that it made no difference inside the component. If that's the case, maybe it should always be a string to be consistent.

Reply:
If there is no need to mark a missing value vs. an empty value then yes, that would work too.

Reply:
I think it depends, but I think it is easier to just have the value be a string, so that when you pass it to other components, you don't have to check if it is undefined. I updated it in my recent push.

Reply:
Are you sure you pushed it? It's still possible to return `null`

Reply:
@FelipeTrost ping